app.rb

```diff
@@ -24,8 +28,12 @@ def save_files
   def recover_files
     genre_file = get_file('./data/genre_list.json')
     music_album_file = get_file('./data/music_album_list.json')
+    game_file = get_file('./data/game.json')
+    author_file = get_file('./data/author.json')
```

Comment:
Good job adding the file, I see that you have a typo in the path of the author file. Kindly add 's' to the author like this: 
`author_file = get_file('./data/authors.json')`